tests/helpers/dom.js

```diff
@@ -154,18 +154,17 @@ function createMockEvent(eventName, element, options={}) {
   return event;
 }
 
-function triggerDelete(editor, direction=DIRECTION.BACKWARD) {
+function triggerDelete(editor, direction=DIRECTION.BACKWARD, options={}) {
```

Comment:
Can we document what `options` is expected to be here? I presume `{ctrlKey: true}` and such.

Reply:
👍